test/bin/common.sh

```diff
@@ -123,11 +138,10 @@ get_vm_bridge_ip() {
     # 10: virbr0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN group default qlen 1000
     #     inet 192.168.122.1/24 brd 192.168.122.255 scope global virbr0
     #        valid_lft forever preferred_lft forever
-
     ip -f inet addr show "${bridge}" | grep inet | awk '{print $2}' | cut -d/ -f1
 }
 
 # The IP address of the current host on the bridge used for the
 # default network for libvirt VMs.
 # shellcheck disable=SC2034  # used elsewhere
-VM_BRIDGE_IP="$(get_vm_bridge_ip)"
+VM_BRIDGE_IP="$(get_vm_bridge_ip "default")"
```

Comment:
This var is used in `scenario.sh`: `WEB_SERVER_URL="http://${VM_BRIDGE_IP}:${WEB_SERVER_PORT}"`.
Does it mean "isolated network" scenarios will be installed from an "all-in-one ISO" containing installer+commit+embedded images? (Just like the `scripts/image-builder/`). If yes, looks like we'll have another blueprint group (or atleast group "2.5"):
- `rhel-92` edge-commit
- `rhel-92 + microshift + images` edge-commit
- `all-in-one ISO` edge-installer

It cannot be without an effect on how long `ci_phase_iso_build.sh` takes to complete (buildiong edge-installers is kind of slow...).
Could we use "regular" install procedure just change the network post install?
I'd say we don't care how the system gets installed, but we care about the environment when it boots

Reply:
We decided to split the prerequisite work from the implementation, so I see it can be confusing. 
What's going to happen in the implementaton scenario file is the following.
```
# Redefine network-related settings to use the isolated network bridge
VM_BRIDGE_IP="$(get_vm_bridge_ip "${VM_ISOLATED_NETWORK}")"
# shellcheck disable=SC2034  # used elsewhere
WEB_SERVER_URL="http://${VM_BRIDGE_IP}:${WEB_SERVER_PORT}"

scenario_create_vms() {
    prepare_kickstart host1 kickstart.ks.template rhel-9.2-microshift-source
    # Use the isolated network when creating a VM
    launch_vm host1 "" "${VM_ISOLATED_NETWORK}"
    verify_no_internet "host1"
}
```

Note that I'm providing the network argument to the `launch_vm` function, so it's going to use the same ISO, just different boot parameters. I hope it answers your questions.